plasmapy/formulary/parameters.py

```diff
@@ -533,16 +540,141 @@ def ion_sound_speed(
 """Alias to `~plasmapy.formulary.parameters.ion_sound_speed`."""
 
 
-# This dictionary defines coefficients for thermal speeds
-# calculated for different methods and values of ndim.
-# Created here to avoid re-instantiating on each call
-_coefficients = {
-    1: {"most_probable": 0, "rms": 1, "mean_magnitude": 2 / np.pi},
-    2: {"most_probable": 1, "rms": 2, "mean_magnitude": np.pi / 2},
-    3: {"most_probable": 2, "rms": 3, "mean_magnitude": 8 / np.pi},
-}
+def thermal_speed_coefficients(method: str, ndim: int) -> float:
+    r"""
+    Get the appropriate coefficient for calculating the thermal speed :math:`v_{th}`
+    based on the given ``method`` and ``ndim``.  (See the
+    `~plasmapy.formulary.parameters.thermal_speed` :ref:`Notes <thermal-speed-notes>`
+    section for further details.)
```

Comment:
```suggestion
    based on the given ``method`` and ``ndim``.  
    
    See the `~plasmapy.formulary.parameters.thermal_speed` 
    :ref:`Notes <thermal-speed-notes>` section for further details.)
```
[optional]

Reply:
I reworked this part a little.  I wasn't happy with how it was sounding.